products/cloudrun/api.yaml

```diff
@@ -270,6 +265,25 @@ objects:
     required: true
     description: Spec holds the desired state of the Service (from the client).
     properties:
+    - !ruby/object:Api::Type::Array
+      name: traffic
+      description: |-
+        Traffic specifies how to distribute traffic over a collection of Knative Revisions and Configurations
+      item_type: !ruby/object:Api::Type::NestedObject
+        properties:
+        - !ruby/object:Api::Type::String
+          name: revisionName
+          description: |-
+            RevisionName of a specific revision to which to send this portion of traffic.
+        - !ruby/object:Api::Type::Integer
+          name: percent
+          required: true
+          description: |-
+            Percent specifies percent of the traffic to this Revision or Configuration. This defaults to zero if unspecified.
```

Comment:
A bunch of these yaml lines are one line, which I'm not against (the Terraform descriptions look less terrible then) but it doesn't seem consistent?

Reply:
Ah these were added by hand whereas all other fields were added by script. I also dislike the multi line descriptions but I'll update for consistency 👍